src/main/java/jarvis/commands/DeleteCommand.java

```diff
@@ -0,0 +1,45 @@
+package jarvis.commands;
+
+import jarvis.Storage;
+import jarvis.Ui;
+import jarvis.exceptions.InvalidIndexException;
+import jarvis.tasks.Task;
+import jarvis.tasks.TaskList;
+
+/**
+ * Represents a command to delete a task in the Jarvis app.
+ */
+public class DeleteCommand implements Command {
+
+    private String userInput;
+
+    public DeleteCommand(String userInput) {
+        this.userInput = userInput;
+    }
+
+    /**
+     * Executes the delete command by removing the specified task from the task
+     * list.
+     *
+     * @param taskList The TaskList containing the tasks.
+     * @param ui       The Ui for user interface interactions.
+     * @param storage  The Storage for saving tasks.
+     * @throws InvalidIndexException If an invalid index is provided.
+     */
+    @Override
+    public void execute(TaskList taskList, Ui ui, Storage storage) throws InvalidIndexException {
+        String[] userInputSpilt = userInput.split(" ");
```

Comment:
Same here. I don't know whether `userInputSplit` is a good name for a variable since it sounds like a verb (method).